drivers/CoreQSPI/tests/CoreQSPI_test.cpp

```diff
@@ -143,9 +143,9 @@ TEST(CoreQSPITest, write)
 {
 	auto coreqspi = CoreQSPI {};
 
-	char data[] = "This is a test";
+	uint8_t data[] = "This is a test";
 
-	auto bytes_written = coreqspi.write(0x2A, -1, 0x2B, data);
+	auto bytes_written = coreqspi.write(0x2A, -1, 0x2B, data, sizeof(data));
```

Comment:
ici c'est plutôt `std::size()` que tu veux parce que `sizeof()` va te retourner la taille occuper par la mémoire, pas la taille du tableau. dans ce cas c'est la même chose parce que tu es sur `uint8_t` mais c'est risqué de parier dessus à chaque fois, il faudrait faire `(sizeof data) / (sizeof data[0])` pour que ce soit juste.

- https://en.cppreference.com/w/cpp/language/sizeof
- https://en.cppreference.com/w/cpp/iterator/size

```suggestion
	auto bytes_written = coreqspi.write(0x2A, -1, 0x2B, data, std::size(data));
```

Reply:
Oui effectivement, on peut utiliser `sizeof` :)